.gitlab-ci.yml

```diff
@@ -88,3 +89,20 @@ deploy-fallback-only:
   extends: .kubectl
   script:
     - ./deploy/deploy.sh HX # fallback
+
+lightouse:
+  image: cypress/browsers:node14.15.0-chrome86-ff82
+  only:
+    refs:
+      - main
+  stage: lighthouse
+  before_script:
+    - git --version
+    - node --version
+    - npm --version
+    - yarn --version
+    - yarn
+  script:
+    # Could we use the artifacts from before instead of rebuilding?
```

Comment:
Yeah, you're right. From the `build_static_assets` job:
```
  artifacts:
    paths:
      - build/
```
`build` directory will have it there